### DISCUSSION TITLE
Cloned Project, how to get started?

### DISCUSSION THREAD

**Discussion by flamecopper** [facebook/docusaurus#8990] [Q&A]
Hi there,

I was trying to run the following command from a clone project, however, it seems like there is an error. Do advise?
npm start run

> xxx-training@0.0.0 start
> docusaurus start run

sh: 1: docusaurus: not found

**Comment by Josh-Cena** [comment]
Have you run `npm install`? Also the command is `npm run start`. This has been mentioned in every getting started guide I believe.
